README.md

```diff
@@ -30,6 +30,46 @@ $ make build-docker-image #Builds the image, will only have to be re-run if you
 $ make docker #Runs the image
 ```
 
+## Using with your own Myradio
+
+### Inital setup
+
+Next you need a api_key to allow the website to access myradio's show information,
+
+login into database with details used during setup of myradio
+
+```sql
+INSERT INTO myury.api_key (key_string, description) VALUES ('ARANDOMSTRINGOFCHARACTERS', '2016-site development api key');
+INSERT INTO myury.api_key_auth (key_string, typeid) VALUES ('ARANDOMSTRINGOFCHARACTERS', (SELECT typeid FROM l_action WHERE phpconstant = 'AUTH_APISUDO'));
+```
+
+[please choose a better key than 'ARANDOMSTRINGOFCHARACTERS']
```

Comment:
square brackets is markdown syntax - and you should probably give an example of generating a random key like `openssl rand -hex 32` - which should come before the SQL statements

Reply:
bump